### PR TITLE
Fix images not loading in mutiple datasources case

### PR DESCRIPTION
**Pull request by mateusfreira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,23 +16,23 @@ Note - The official name for the service is "AWS HealthImaging".  Before GA it w
 # AWS_HOST
 docker run -p 8089:8089 -e AWS_ACCESS_KEY_ID='YOUR_KEY' -e AWS_SECRET_ACCESS_KEY='YOUR_SECRET' -e AWS_REGION='YOUR_REGION' flexview/ohif-aws-healthimaging-proxy
 ```
-* Add healthlake adapter as an OHIF plugin `platform/viewer/pluginConfig.json`
+* Add healthimaging as a dependency `yarn add  ohif-aws-healthimaging --ignore-workspace-root-check`
```
Comment (mateusfreira) [inline README.md:19]: Small fixes in the docs

File: src/createDicomWebTreeApi.ts [hunk 2]
```diff
@@ -529,6 +533,22 @@ function createDicomWebTreeApi(dicomWebConfig, UserAuthenticationService) {
       });
       return imageIds;
     },
+    getStudyInstanceUIDs({ params, query }: { params: any; query: any}) {
```
Comment (mateusfreira) [inline src/createDicomWebTreeApi.ts:536]: This method was introduced in the newer version of OHIF and I am adding it here just to be compatible with latest v3

**Review by IbrahimCSAE — APPROVED** [review]

**Comment by IbrahimCSAE** [comment]
LGTM

File: src/createDicomWebTreeApi.ts [hunk 3]
```diff
@@ -43,6 +43,10 @@ const initializeHealthlakeFetch = (healthlake) => {
       const datastoreId = urlParams.get('DatastoreID');
       const collectionId = urlParams.get('ImageSetID');
       const imageFrameId = urlParams.get('frameID');
```
Comment (IbrahimCSAE) [inline src/createDicomWebTreeApi.ts:45]: would the url contain healthlake or healthimaging?

**Pull request merged**